src/selectors/wallet.js

```diff
@@ -83,6 +83,22 @@ export const getSlicesWithLastUsed = createSelector(
   }
 );
 
+/**
+ * Gets the set of spendable slices, i.e. ones that don't have a
+ * pending utxo and are not spent
+ */
+export const getSpendableSlices = createSelector(
```

Comment:
We should be able to use this in other places in the code too. I.e. there was a change requested that pending slices should not be spendable. This would allow us to populate tables only with non-pending, i.e. "spendable" slices.